Atomic/syscontainers.py

```diff
@@ -1821,6 +1821,28 @@ def _check_system_oci_image(self, repo, img, upgrade):
         current_rev = repo.resolve_rev(imagebranch, True)
         if not upgrade and current_rev[1]:
             return False
+
+        can_use_skopeo_copy = str(util.check_output([util.SKOPEO_PATH, "copy", "--help"])).find("ostree") >= 0
```

Comment:
``util.check_output()`` forwards to ``subprocess.check_output()``. On Python 2 the result will be a string but on Python 3 it will be bytes. While one can still technically find ``"ostree"`` in the results, it will be looking at a string representation of bytes.

To check the same result on both Python 2 and 3 you can use ``decode()`` like:

```python
can_use_skopeo_copy = util.check_output([util.SKOPEO_PATH, "copy", "--help"]).decode().find("ostree") >= 0
```

In both cases it will be checking a ``unicode`` instance.

Reply:
thanks, I've pushed a fixup patch to add the missing `.decode()`